src/svg-renderer.js

```diff
@@ -210,13 +212,13 @@ class SvgRenderer {
     }
 
     /**
-     * @param {string} tagName svg tag to search for
+     * @param {string} [tagName] svg tag to search for
      * @return {Array} a list of elements with the given tagname in _svgTag
      */
-    _collectElements (tagName) {
+    _collectElements (tagName = null) {
```

Comment:
Why add collecting tagName = null elements here?

Reply:
If a tag name is not passed, it will default to `null`, which line 221 checks. I can remove this and just check if `typeof tagName === undefined`.